docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+    
+| Main App          | Possible Sub App Combinations |
+|-------------------|-------------------------------|
+| Native   Main App | Native + Dotnet               |
+| Dotnet   Main App | Dotnet + Native               |
+| Web   Main App    | Web + Native +   Dotnet       |
+
+- CLI
+  - Added support for importing Hybrid and Multi projects
+- VSCode
+  - Added support for importing Hybrid and Multi projects. Developer can import a Tizen.Native/Tizen.WEB/TizenDotnet Project to VSCode and then can build and pakcage.
+  - Added support for importing wgt(Tizen Web application package file) into Visual Studio Code Extension for Tizen
+- Visual Studio(Windows)
+  - Added support for importing Hybrid and Multi projects. Developer can import a Tizen.Native/Tizen.WEB/TizenDotnet Project to VS and then can build and pakcage.
+  - Added support for importing wgt(Tizen Web application package file) into Visual Studio Extension for Tizen
+
+
+### Fixed Bugs
+  - Fixed the issue in API Checker due to which warnings were not displayed for web apps in VS extension for Tizen.
+  - Fixed the error that occured on selecting TV profile while creating a new project in VS extension for Tizen.
+  - Fixed the failure to open yaml config files for native apps in VS extension for Tizen. 
+  - Fixed the issue due to which there was no option to import Tizen TV web app projects in VS & VSCode extension for Tizen.
+  - Fixed the issue in Import functionality and added missing support to import wgt(Web Application Package file) in VS & VSCode extension for Tizen
+
+### Deprecated
+  - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+
+
+### Known Issues
+
+- Installer
+  - In macOS, Tizen Studio related file are marked as Malware, hence it is recommended to select **Anywhere** under allow apps as described in https://www.imore.com/how-open-apps-anywhere-macos-catalina-and-mojave
+- Tizen Studio
+  - Valgrind profiling is not supported in Tizen 6.5 platform
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - In macOS from catalina and above versions, Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for **C:\ProgramFiles** location administrator privilege is needed to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you if you try to install it into such a directory.
```

Comment:
The first line seems incomplete, so I have made suggestions to make it understandable. Please let me know if this delivers the meaning properly. 
“The” has been removed before “Tizen SDK tools”, “Tizen Installer”, "Baseline SDK Installer" since it is recommended to not use “the” before nouns such as tool names. 

```suggestion
  - If you install Tizen Studio in a directory that requires administrator privileges for access, an alert will be given. . For example, for **C:\ProgramFiles** location administrator privilege is needed to run Tizen SDK tools. Tizen Installer and Baseline SDK Installer alerts you if you try to install it into such a directory.
```